source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -105,7 +105,15 @@ After the record is created wait for up to an hour for the CloudFront distributi
 
 #### CDN Configuration Options
 
```

Comment:
I agree. I've swapped the subheading for a short paragraph explaining what the table is for.

> ... and then a section here explaining how tenants should use the table to choose the right option for their needs. We should explicitly state here that more info on each can be found in the relevant sections below.

I haven't done this because the table doesn't help users do that, but added the reference to the subsections so they know what to read to get the useful information.